inputfiles/addedTypes.jsonc

```diff
@@ -354,6 +354,10 @@
                         "decoding": {
                             "name": "decoding",
                             "overrideType": "\"async\" | \"sync\" | \"auto\""
+                        },
+                        "loading": {
+                            "name": "loading",
```

Comment:
```suggestion
```

`name` is actually not needed here, same for above items (you don't need to fix them here)